azure-toolkit-libs/azure-toolkit-sqlserver-lib/src/main/java/com/microsoft/azure/toolkit/lib/sqlserver/model/SqlFirewallRuleEntity.java

```diff
@@ -14,7 +15,7 @@
 @SuperBuilder(toBuilder = true)
 public class SqlFirewallRuleEntity implements IAzureResourceEntity {
 
-    public static final String ACCESS_FROM_LOCAL_FIREWALL_RULE_NAME = "ClientIPAddress_" + NetUtils.getHostName() + "_" + NetUtils.getMac();
+    public static final String ACCESS_FROM_LOCAL_FIREWALL_RULE_NAME = "ClientIPAddress_" + NetUtils.getHostName() + "_" + MacUtils.getMac();
```

Comment:
Do we need to make it a method instead of a static field?

Reply:
I'd like not. will test it in mac to avoid same issue.
and, how about let us consider to refactor it in mysql refactor task?